cfgov/jinja2/v1/_includes/organisms/post-preview.html

```diff
@@ -39,13 +39,117 @@
 
    ========================================================================== #}
 
-{% macro render(settings, path) %}
-
 {% import 'macros/share.html' as share with context %}
 {% import 'category-slug.html' as category_slug %}
 {% import 'macros/time.html' as time %}
 {% from 'macros/util/format/url.html' import location_image_url as location_image_url %}
 
+{% if value %}
+    <article class="o-post-preview">
+        <div class="meta-header">
+        <span class="date meta-header_right">
+            Published {{ time.render(value.published_date, {'date':true}) }}
+        </span>
+            {% if value.post_category_1 %}
+                {% set use_post_category = true %}
+                {{ category_slug.render(value.post_category_1,
+                        value.path,
+                        'post_slug meta-header_left',
+                        use_post_category) }}
+            {% endif %}
+            {% if value.post_category_2 %}
+                |
+                {{ category_slug.render(value.post_category_2,
+                        value.path,
+                        'post_slug meta-header_left',
+                        use_post_category) }}
+            {% endif %}
+        </div>
+        {% if value.event.start_date %}
+            <div class="o-post-preview_image-container">
+                <time class="calendar-icon"
+                      data-month="{{ value.event.start_date | date('%b') }}"
+                      data-day="{{ value.event.start_date | date('%-d') }}"
+                      datetime="{{ value.event.start_date | date('%c') }}">
+                    <span class="u-visually-hidden">{{ value.event.start_date| date('%b %-d, %Y') }}</span>
+                </time>
+                {% if value.event.stream_link %}
+                    <img class="o-post-preview_image"
+                         src="http://placehold.it/160x90"
+                         alt="Livestream event">
+                {% else %}
+                    <img class="o-post-preview_image"
+                         src="{{ location_image_url({
+                                 'location': value.event.city
+                                 | default('Washington, DC', true)
+                                 | urlencode,
+                                 'zoom':     '12',
+                                 'scale':    '2',
+                                 'size':     '276x155'
+                                 }) }}"
+                         alt="Google Maps image of {{ value.event.venue }}">
+                {% endif %}
+            </div>
+        {% elif value.image.url %}
+            <div class="o-post-preview_image-container">
+                <img class="o-post-preview_image"
+                     src="{{ value.image.url }}"
+                     alt="{{ value.image.alt if value.image.alt else '' }}">
+            </div>
+        {% endif %}
```

Comment:
There's CSS to make the container full-width in cases where there is no image or date, like newsroom posts and notices.

![image](https://cloud.githubusercontent.com/assets/1860176/11378111/a67aa21a-92b7-11e5-81d6-180dc9dcb5f9.png)